build_tools/build_detect_platform

```diff
@@ -668,6 +668,23 @@ else
   fi
 fi
 
+if test -n "`echo $TARGET_ARCHITECTURE | grep ^ppc64`"; then
+  # check for GNU libc on ppc64
+  $CXX -x c++ - -o /dev/null 2>/dev/null  <<EOF
+    #include <stdio.h>
+    #include <stdlib.h>
+    #include <gnu/libc-version.h>
+
+    int main(int argc, char *argv[]) {
+      printf("GNU libc version: %s\n", gnu_get_libc_version());
+      return 0;
+    }
+EOF
+  if [ "$?" != 0 ]; then
+      PPC_LIBC_IS_GNU=0
+  fi
+fi
```

Comment:
Should this be LIBC_IS_GNU and be on by default?  I am missing where this value is set to 1 (non-zero).

Reply:
Nope. It is designed so that it checks that `PPC_LIBC_IS_GNU` is not-equal-to `0` in the default case. If the flag is absent then it is not-equal-to `0`. So we only need to set it in this specific case